### PR TITLE
[Issue-17] Add project using webpack-file-preprocessor-plugin to minify javascript <Anusha>

### LINKED ISSUE #17: Add an example project using the plugin to minify javascript
Add an example project using the plugin to minify javascript.

**Requirements:**
- Take reference from the existing example under `examples/` directory. Use similar conventions and code style.
- Use `yarn` instead of `npm`.
- Use any popular javascript minifier.

### PR DESCRIPTION
**Pull request by anusha5695**
**Issue:**
This PR is related to the issue: https://github.com/kabirbaidhya/webpack-file-preprocessor-plugin/issues/17
Resolves #17 

**Summary of changes:**
1. Added uglify-js package for minifying JS
2. Create example project with js code
3. Used the plugin in webpack config

### CODE REVIEW

**Comment by anusha5695** [comment]
@kabirbaidhya : Done 👍

File: examples/js-minifier/src/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+   * Add two numbers.
+   * 
+   * @param {number} first The first number.
+   * @param {number} second The second number.
+   * @returns {number} The sum of the two numbers.
+**/
```
Comment (kabirbaidhya) [inline examples/js-minifier/src/index.js]: JSDoc dosesn't look right to me. You'll need to remove the leading spaces a bit. 

Something like these:
https://github.com/kabirbaidhya/webpack-file-preprocessor-plugin/blob/a363762b97f2e56579de9a71b62974d8b739fabe/src/util.js#L21-L47

Please refer to this for more info - https://eslint.org/docs/rules/valid-jsdoc

**Comment by kabirbaidhya** [comment]
@anusha5695 Everything else looks fine. Please resolve the suggested comments and I can sign off. Thanks!

File: examples/js-minifier/src/index.js (hunk 1 above)
Comment (anusha5695) [reply examples/js-minifier/src/index.js]: Done 👍

**Comment by kabirbaidhya** [comment]
Hey @anusha5695 - just a suggestion. Looks like your contributions aren't being shown in GitHub even though 3 of your PRs have been merged in this repo.

It's perhaps because your GitHub email address isn't properly configured - check if [this thread](https://stackoverflow.com/questions/15289768/github-commits-arent-recorded-in-the-your-contributions-calendar#answer-31007670) can do any help. 

image

**Pull request merged**